universal-login-commons/test/integration/ethereum/validators/hasEnoughToken.ts

```diff
@@ -15,7 +16,7 @@ describe('INT: hasEnoughToken', async () => {
   const gasPrice = '2';
   let token: Contract;
   let wallet: Wallet;
-  const createPaymentOptions = (gasToken: string, gasLimit: utils.BigNumberish, gasPrice: utils.BigNumberish) => ({gasToken, gasLimit, gasPrice});
+  const createPaymentOptions = (gasToken: string, gasLimit: utils.BigNumberish, gasPrice: utils.BigNumberish, refundReceiver: string) => ({gasToken, gasLimit, gasPrice, refundReceiver});
```

Comment:
For purpose of this test helper method `createPaymentOptions` can always return `refundReceiver == 0x0...0`. No need to add new parameter.